tests/integration/Core/Framework/KernelTest.php

```diff
@@ -17,28 +16,27 @@ class KernelTest extends TestCase
 {
     use KernelTestBehaviour;
 
-    /**
-     * @deprecated tag:v6.7.0 - remove
-     */
-    public function testDatabaseTimeZonesAreEqual(): void
+    public function testMySQLSessionDefaultsAreNotSet(): void
     {
-        $env = (bool) EnvironmentHelper::getVariable('SHOPWARE_DBAL_TIMEZONE_SUPPORT_ENABLED', false);
+        $_SERVER['SQL_SET_DEFAULT_SESSION_VARIABLES'] = '0';
```

Comment:
you should use `\Shopware\Core\Framework\Test\TestCaseBase\EnvTestBehaviour` so, the variables are reset correctly after the test and do not interfere